### DISCUSSION TITLE
Time to Customize Bromite

### DISCUSSION THREAD

**Discussion by blogmangit** [bromite/bromite#2000] [General]
@csagan5, So long you have maintained the concept of keeping bromite same as chromium no user interface customization & I truly appreciate that. But to my surprise you have made a big customization that is move addressbar to bottom. So I think it would be better to ultimately break the stereotype & also apply extension patch to bromite from kiwibrowser source repo. It would be great. 

Regards.

**Comment by csagan5** [comment]
> you have maintained the concept of keeping bromite same as chromium no user interface customization

That has never been the case, see https://github.com/bromite/bromite/blob/master/README.md#features-not-related-to-privacy
